README.md

```diff
@@ -18,28 +18,28 @@ The template is created by [GDG Jalandhar](https://meetup.com/GDG-Jalandhar/) te
 | **Fast and optimized** | 100/100 PWA on Lighthouse |
 | **Works offline** | Can works offline |
 | **Mobile first** | Mobo Friendly Web app can be installed as a native app on your phone |
-| **SEO optimized** | index all content and get to the top in search results |
-| **Speakers and schedule management** | keep and update all information in the JSON File |
+| **SEO optimized** | Index all content and get to the top in search results |
+| **Speakers and schedule management** | Keep and update all information in the JSON File |
 | **Registration** | Attendees can register for the event using Firebase Realtime Database |
 
 
 ## Getting Started
 1. [Fork repository](https://github.com/Vrijraj/gdgx-event-web-app/fork) and clone it locally
-1. Setup Environment
+2. Setup Environment
```

Comment:
What is the reason for changing these? Adding specific numbers like this makes things more fragile. Markdown can process them and assign the proper numbers like in the current readme.